.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,21 @@
 <!--
-Update "[ ]" to "[x]" to check a box
+Before submitting a PR, please read https://github.com/tauri-apps/tauri/blob/dev/.github/CONTRIBUTING.md#pull-request-guidelines
 
-Please make sure to read the Pull Request Guidelines: https://github.com/tauri-apps/tauri/blob/dev/.github/CONTRIBUTING.md#pull-request-guidelines
--->
-
-### What kind of change does this PR introduce?
-<!-- Check at least one. If you are introducing a new binding, you must reference an issue where this binding has been proposed, discussed and approved by the maintainers. -->
-
-- [ ] Bugfix
-- [ ] Feature
-- [ ] Docs
-- [ ] New Binding issue #___
-- [ ] Code style update
-- [ ] Refactor
-- [ ] Build-related changes
-- [ ] Other, please describe:
+1. Give the PR a descriptive title.
 
-### Does this PR introduce a breaking change?
-<!-- If yes, please describe the impact and migration path for existing applications in an attached issue. -->
+  Examples of good title:
+    - fix(windows): fix race condition in event loop
+    - docs: update docstrings
+    - feat: add `Window::set_fullscreen`
 
-- [ ] Yes, and the changes were approved in issue #___
-- [ ] No
+  Examples of bad title:
+    - fix #7123
+    - update docs
+    - fix bugs
 
-### Checklist
-- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
-- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
-- [ ] I have added a convincing reason for adding this feature, if necessary
-
-### Other information
+2. If there is a related issue, referenced in the PR text, e.g. closes #123.
+3. If this change requires a new version, then add a change file in `.changes` directory with the appropriate bump, see https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md
+4. Ensure that all your commits are signed https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits
+5. Ensure `cargo test` and `cargo clippy` passes.
+6. Open as a draft PR if your work is still in progress.
```

Comment:
Not quite sure about this one, i feel like there's missing a word idk.
Maybe `"Choose a draft PR if your work is still in progress."` or `"Propose your changes as a draft PR if your work is still in progress."`

Reply:
"Propose your changes as a draft PR if your work is still in progress." thumbs up